app/src/main/java/com/android/calculator2/widget/CalculatorWidget.java

```diff
@@ -26,20 +27,20 @@
 
 public class CalculatorWidget extends AppWidgetProvider {
     public final static String PREFERENCE_WIDGET_PREAMBLE = "com.android.calculator2.CALC_WIDGET_VALUE_";
-    public static final String DIGIT_0 = "com.android.calculator2.0";
-    public static final String DIGIT_1 = "com.android.calculator2.1";
-    public static final String DIGIT_2 = "com.android.calculator2.2";
-    public static final String DIGIT_3 = "com.android.calculator2.3";
-    public static final String DIGIT_4 = "com.android.calculator2.4";
-    public static final String DIGIT_5 = "com.android.calculator2.5";
-    public static final String DIGIT_6 = "com.android.calculator2.6";
-    public static final String DIGIT_7 = "com.android.calculator2.7";
-    public static final String DIGIT_8 = "com.android.calculator2.8";
-    public static final String DIGIT_9 = "com.android.calculator2.9";
+    public static final String DIGIT_0 = "com.android.calculator2.zero";
```

Comment:
To make it easier to expand the software to more languages in the future, maybe it would be best to keep using 1, 2, 3, ... instead of one, two, three, ..?